virt/virtual_machines/virt-configuring-higher-vm-workload-density.adoc

```diff
@@ -0,0 +1,30 @@
+:_mod-docs-content-type: ASSEMBLY
+[id="virt-configuring-higher-vm-workload-density"]
+= Configuring higher VM workload density
+include::_attributes/common-attributes.adoc[]
+:context: virt-configuring-higher-vm-workload-density
+
+toc::[]
+
+
+:FeatureName: Configuring higher workload density
+include::snippets/technology-preview.adoc[]
+
+To save resources and to increase the number of VMs, You can configure a higher VM workload density in your cluster by over-committing memory resources (RAM).
```

Comment:
Rewrite the sentence to:
To increase the number of VMs per node, you can configure a higher VM workload density in your cluster by over-committing memory resources (RAM) with swap.